### PR TITLE
created an awesome new feature

### PR DESCRIPTION
**Pull request by growthlabs**

### CODE REVIEW

**Comment by madolll** [comment]
looks great

**Comment by ppena11** [comment]
I dont see what you did...

File: changes.js [hunk 1]
```diff
@@ -1 +1,2 @@
-// some changes
\ No newline at end of file
+// some changes
```
Comment (madolll) [inline changes.js:1]: test

**Comment by alineporciuncula** [comment]
Hey

File: changes.js (hunk 1 above)
Comment (ppena11) [inline changes.js:1]: Hwat is this...

Comment (stephcathoh) [reply changes.js:1]: Did you figure it out?

Comment (growthlabs) [reply changes.js:1]: No

Comment (ppena11) [reply changes.js:1]: Ok i see it

File: changes.js [hunk 2]
```diff
@@ -1 +1,4 @@
...
+//SOME MORE CHENGES
```
Comment (growthlabs) [inline changes.js:2]: ergergrgdrgffhfghfgh

**Review by growthlabs — COMMENTED** [review]
I did it. Done. Super.

**Pull request merged**